src/dmd/statementsem.d

```diff
@@ -53,7 +53,7 @@ import dmd.statement;
 import dmd.target;
 import dmd.tokens;
 import dmd.typesem;
-import dmd.visitor;
+import dmd.visitors : Visitor;
```

Comment:
> Out of curiosity why selective import here?

I thought it's a good practice to show which parts are used.
I wasn't aware of this, but as long as some have it fixed, it will trigger errors.
Anyways, this has now been replaced with `dmd.visitor.visitor` and `dmd.visitor.stoppable`.